src/components/ClusterPage/index.jsx

```diff
@@ -117,7 +117,13 @@ class ClusterPage extends React.Component {
           <ClustersList newClusterAdded={isAdded} />
         </div>
         <div className="FooterRow">
-          <p>Copyright © 2020 Crane Cloud. All Rights Reserved.</p>
+          <p>
+            Copyright ©
+            {' '}
```

Comment:
use string interpolation here, instead of 3 expressions: `Copyright ${new Date().getFullYear()} Crane...`

Reply:
@kawerewagaba we could do a sync on this and I swear to you, that's all ESLint!

Reply:
your linter is dumb, you should impart some sense in it - override the rule